packages/aws-cdk/lib/init-templates/v2/sample-app/java/pom.template.xml

```diff
@@ -7,7 +7,7 @@
     <version>0.1</version>
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <cdk.version>%cdk-version%</cdk.version>
+        <cdk.version>%cdk-version%.DEVPREVIEW</cdk.version>
```

Comment:
I thought we were following this pattern - https://github.com/aws/aws-cdk-rfcs/blob/master/text/0079-cdk-2.0.md#releasing-to-package-managers

We should'nt be suffixing DEVPREVIEW to one that already has the word 'alpha' in it.